.github/workflows/tests_secondary.yml

```diff
@@ -27,13 +27,13 @@ jobs:
       fail-fast: false
       matrix:
         browser: [chromium, firefox, webkit]
-        os: [ubuntu-20.04, ubuntu-22.04]
+        os: [ubuntu-20.04]
```

Comment:
Note: This is the same as https://github.com/microsoft/playwright/blob/0717b386f99f9dc56e3241bf665071216e4bf66a/.github/workflows/tests_primary.yml#L29-L67 if I understand it correctly, removing it by that and only testing the actual de-duplicated part.